vulnerablecode/urls.py

```diff
@@ -26,7 +26,7 @@
 
 from rest_framework.routers import DefaultRouter
 
-from vulnerabilities.api import PackageViewSet
+from api.views import PackageViewSet
```

Comment:
> Also could it make sense to have all the sources under an src directory (that's a tad less common for Django)?

I'm sure it's possible to move the entire Django project into a directory called `src`. There wouldn't be much left in the top-level dir of the repository, but maybe that will change over time.
If you want to have i.e. `manage.py` outside of a `src` directory that might require some more fiddling but should be doable as well.
What benefits do you see in having a `src` dir?

> api feels like it is too generic a module name.... What could be more discriminant?

The scope of the name is all apps in the Django project `vulnerablecode`. As long as there is no ambiguity when talking about "the API of vulnerablecode" I think short, generic names are fine. It would be different if we wanted to make the `api` app reusable in other Django projects or turn vulnerablecode into a library.
Having said that, I don't insist on the name at all but I can't think of a better one right now either.

> IMHO we need a top level module

To have it mentioned in `import` statements? Or what would be the benefit? I think we shouldn't stray too far from what Django expects in terms of project structure. I remember reading somewhere about a "flat structure" for Django projects without the individual apps a long time ago. However that sounds like it would be less nesting, not more.